src/views/AffiliatesBanner.tsx

```diff
@@ -48,25 +52,30 @@ export const AffiliatesBanner = ({ withClose = false }: { withClose?: boolean })
   });
 
   const description = (
-    <>
-      {stringGetter({
-        key: STRING_KEYS.REFER_FOR_DISCOUNTS_FIRST_ORDER,
-        params: {
-          AMOUNT_USD: `$${AFFILIATES_FEE_DISCOUNT_USD.toLocaleString()}`,
-        },
-      })}{' '}
-      <br />
-      {stringGetter({
-        key: STRING_KEYS.WANT_TO_VIEW_EARNINGS,
-        params: {
-          LINK: (
-            <Link href={affiliateProgram} isInline isAccent>
-              {stringGetter({ key: STRING_KEYS.AFFILIATES_PROGRAM })} →
-            </Link>
-          ),
-        },
-      })}
-    </>
+    <div tw="flex flex-col">
+      <div>
+        {stringGetter({
+          key: STRING_KEYS.REFER_FOR_DISCOUNTS,
+          params: {
+            AMOUNT_USD: `${AFFILIATES_FEE_DISCOUNT_USD.toLocaleString()}`,
+          },
+        })}
+      </div>
+      {showLink && (
+        <div>
+          {stringGetter({
+            key: STRING_KEYS.WANT_TO_VIEW_EARNINGS,
+            params: {
+              LINK: (
+                <Link isNewPage={false} href="/affiliates" isInline isAccent>
```

Comment:
We should update the `<Link>` component to use `{ Link } from 'react-router-dom'` when we want them to navigate within our router

Reply:
then we can get rid of the `isNewPage` prop

Reply:
also use enum `AppRoute.Affiliates`